TsRandomizer/Randomisation/BestiaryManager.cs

```diff
@@ -116,6 +118,8 @@ public static BossAttributes GetBossAttributes(Level level, int bossId)
 						IsFacingLeft = true,
 						ShouldSpawn = false,
 						TileId = (int)EEnemyTileType.RoboKittyBoss,
+						EnemyArgument = 0,
+						Weaknesses = 282574488338689,
```

Comment:
Would strongly reccomend that arbitrary values like this are made into a more human readable format, like a function that returns the appropriate number when you pass in arguments, like psuedocode: weakeness = weakness_lookup(blunt=true, radiant=true) or something similar

(I was going to say enum but I realized that could get complex with multiple weaknesses)

Reply:
well there are flag enums, that you can group together like 
Weakness.Fire | Weakness.Dark